app/_gateway_entities/key-set.md

```diff
@@ -30,5 +25,25 @@ schema:
 
 ---
 
-@todo
-You can assign one or many keys to a JSON Web Key Set. This can be useful to logically group multiple keys to use for a specific application or service. Key Sets are also the preferred way to expose keys to plugins because they tell the plugin where to look for keys or have a scoping mechanism to restrict plugins to just some keys.
\ No newline at end of file
+## What is a Key Set?
+
+A Key Set is a collection of {{site.base_gateway}} [Keys](/gateway/entities/key).
+
+You can assign one or many Keys to a Key Set. This can be useful to logically group multiple Keys to use for a specific application or service. Key Sets allow you to give a plugin access to a specific list of Keys.
+
+Currently, Key Sets can be used with the following plugins:
```

Comment:
```suggestion
Key Sets can be used with the following plugins:
```
Let's remove "currently" since the pages are versioned, so it's always "current", technically.